### PR TITLE
Fix FE web insecure cookie setting #26056

### LINKED ISSUE #26056: [Bug] FE web insecure cookie setting
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

FE web exists insecure cookie setting param

### What You Expected?

FE web exists insecure cookie setting param, need to optimize

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LuGuangming**
## Proposed changes

Issue Number: close #26056

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public ActionAuthorizationInfo checkWithCookie(HttpServletRequest request,
     protected void addSession(HttpServletRequest request, HttpServletResponse response, SessionValue value) {
         String key = UUID.randomUUID().toString();
         Cookie cookie = new Cookie(PALO_SESSION_ID, key);
-        cookie.setSecure(false);
+        cookie.setSecure(true);
```
Comment (jeffreys-cat) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java:107]: `cookie.setSecure(true)` means Cookies will only be passed under the HTTPS protocol, which will make doris UI login can not work.

**Comment by hf200012** [comment]
127.0.0.1 works fine, but you can’t log in with a normal IP address. Don’t merge it yet.

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java (hunk 1 above)
Comment (vinlee19) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java:107]: Maybe  we can set the cookie secure flag in the fe.config ,by default the values is false.

Comment (LuGuangming) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java]: we can use enale_https to set cookie.setSecure true or false

Comment (LuGuangming) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java]: image
test case success

**Review by hf200012 — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
lgtm

**Pull request merged**